pkg/flist/flist.go

```diff
@@ -35,10 +41,19 @@ var (
 	ErrNotMountPoint                   = errors.New("path is not a mountpoint")
 	ErrTransportEndpointIsNotConencted = errors.New("transport endpoint is not connected")
 	ErrZFSProcessNotFound              = errors.New("0-fs process not found")
+	ErrHashNotSupported                = errors.New("hash not supported by flist host")
+	ErrHashInvalid                     = errors.New("invalid hash length")
```

Comment:
I suggest to change it to `ErrInvalidHashLen`, as it could be eg Malformed hash or something else

Reply:
Yeah, will do.